### ISSUE TITLE
Autoteleport after rfc - lowering deposits and fees 

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#10151]
How is it with Autoteleport currently? Deposits on AHP were lowered. Can we have a look at Autoteleport again and maybe lower the requirements? 

![image](https://github.com/kodadot/nft-gallery/assets/90852205/3ace9dd4-f179-4320-b4af-253bdfce1496)
![image](https://github.com/kodadot/nft-gallery/assets/90852205/663714ff-140e-4a9f-bf7e-c1ac39899dd9)
![image](https://github.com/kodadot/nft-gallery/assets/90852205/e57af861-c68c-42ed-b4ef-ef851e511b6a)

looks like it works, and it will leave the account with 0.1 DOT

the existential deposit is way lower now

**Comment by hassnian** [comment]
as suggested https://github.com/kodadot/nft-gallery/issues/9962#issuecomment-2034106409 we introduced 2 sets of ED, if you have to teleport, we will user higher EDs just to be safe

teleport ED:
ahk\rmrk\ksm: 0.00066 KSM
ahp: 0.5 DOT
dot: 1.5 DOT

normal ED:
ahk\rmrk\ksm: 0.00033 KSM
ahp: 0.1 DOT
dot: 1 DOT

see https://github.com/kodadot/nft-gallery/issues/9596#issuecomment-1994597629

![CleanShot 2024-04-27 at 08 31 02@2x](https://github.com/kodadot/nft-gallery/assets/44554284/ad188d20-1207-4525-87bf-ebcb16b33f34)

**Comment by exezbcz** [comment]
@hassnian yes, but the ahp existential deposit was recently changed, IMO its either 0.01 or 0.001 - will check and update you on that
